docker/validator-dynamic/docker-run-dynamic.sh

```diff
@@ -29,6 +29,9 @@ if [ -n "${CFG_SEED_PEER_IP}" ]; then # Seed peer ip for discovery
 	    params+="--bootstrap /ip4/${CFG_SEED_PEER_IP}/tcp/6180 "
 fi
 
-/opt/libra/bin/dynamic-config-builder --data-dir /opt/libra/etc --output-dir /opt/libra/etc/ ${params[@]}
+/opt/libra/bin/validator-config-builder \
+    --data-dir /opt/libra/etc \
```

Comment:
I'm not a fan of `etc` as a folder name, is there a better descriptive name?

Reply:
cc: @opsguy